game.py

```diff
@@ -0,0 +1,139 @@
+import sqlite3
+import os
+from random import choice, randrange
+from flask import request
+from flask_sqlalchemy import SQLAlchemy
+from web_app import app, APP_ROOT
+from sqlalchemy.exc import IntegrityError
+
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///db.sqlite3"
+db = SQLAlchemy(app)
+
+filename_txt = "user" + "_file.txt"
+no_of_selected = 5
+#levels = [0, 1, 2, 0, 0, 1, 0, 0, 0, 1]
+chosen_list = []
+
+class User(db.Model):                       # flask-sglalchemy z class spravi tabulku s tymto menom
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(30), unique=True, nullable=False)
+    words = db.relationship('Word', backref='user', lazy=True)
+
+class Word(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    unknown = db.Column(db.String(100), unique=True, nullable=False)
+    known = db.Column(db.String(100), unique=True, nullable=False)
+    level = db.Column(db.Integer)
+    user_id = db.Column(db.Integer, db.ForeignKey('user.id'), nullable=False)
+
+    def __repr__(self):
+        return self.unknown, self.known
+
+def save_uploaded_file():
+    """saves the uploaded file by user into 'user' directory"""
+    target = os.path.join(APP_ROOT, 'user/')
+    print(target)
+    if not os.path.isdir(target):
+        os.mkdir(target)
+
+    for file in request.files.getlist("file"):
+        print(file)
+        filename = "user_file" + ".txt"
+        destination = "/".join([target, filename])
+        file.save(destination)
+
+def new_words_from_text_file():
+    """appends new words from the text file to the database, if user_textfile.txt uploaded"""
+    db.create_all()
+
+    def add(db):
+        for line in user_file:     
```

Comment:
Funkce ve funkci je občas užitečná, ale je lepší se tomu vyhnout. Dej funkci `add` na stejnou úroveň jako `new_words_from_text_file` a pojmenuj jí `_add`.
https://stackoverflow.com/questions/8689964/why-do-some-functions-have-underscores-before-and-after-the-function-name